try:
    #no error part
    print("no err")
    #error part
    print (10/0)
except:
    print("err")
